### DISCUSSION TITLE
CVE-2023-5072 in org.json/json

### DISCUSSION THREAD

**Discussion by chovyy** [primefaces/community#291] [PrimeFaces] [answered]
Last week, our Dependency Check started to list CVE-2023-5072 for PrimeFaces (here: v10.0.19 + v11.0.12) since it is shading org.json/json version 20230227.

Will you update this in the next releases?

- https://nvd.nist.gov/vuln/detail/CVE-2023-5072
- https://github.com/stleary/JSON-java/issues/789

**Comment by melloware** [comment, marked as answer]
See: https://github.com/primefaces/primefaces/issues/10877  it is slated for 10.0.20 and 11.0.13

**Comment by chovyy** [comment]
Great. Thanks a lot!
